core/src/stored/ndmp_tape.cc

```diff
@@ -357,7 +357,7 @@ static inline bool bndmp_read_data_from_block(JobControlRecord* jcr,
     } else {
       // Read the next block into our buffers.
       if (!ReadNextBlockFromDevice(dcr, &rctx->sessrec, NULL,
-                                   MountNextReadVolume, &ok)) {
+                                   MountNextReadVolume, NULL, &ok)) {
```

Comment:
Shouldn't we use nullptr instead?

Reply:
i would leave it as NULL for now since the rest of the file also uses NULL.